ci/nodepool/nodepool.yaml

```diff
@@ -14,26 +14,31 @@ cron:
   cleanup: '*/1 * * * *'
 
 labels:
+  # unversioned labels used to abstract the distribution version from the job,
+  # useful where the job just wants the latest available fedora, not a specific version.
+  # pulp-smash gets its own label to help nodepool's node launch calculations keep smash
+  # nodes available when they're needed
   - name: pulp-smash-np
-    image: f23-np
+    image: f25-np
     min-ready: 0
     providers:
       - name: cios
     ready-script: fix_hostname.sh
-  - name: f23-np
-    image: f23-np
+  - name: fedora-np
+    image: f25-np
     min-ready: 0
     providers:
       - name: cios
     ready-script: fix_hostname.sh
-  - name: f23-vanilla-np
-    image: f23-vanilla-np
+  # unversioned docker image, similar to fedora-np, but with docker installed
+  - name: docker-np
+    image: docker-np
     min-ready: 0
     providers:
       - name: cios
     ready-script: fix_hostname.sh
-  - name: f23-docker-np
-    image: f23-docker-np
+  - name: fedora-np
```

Comment:
Missed this on the previous review, but it seems to be duplicated. There is a fedora-np above already.